### PR TITLE
Handle Phishing-Alerts

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
In Progress

## Description
An article to provide initial context to the Phishing Alerts pack.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: content-repo/extra-docs/packs/Phishing-Alerts.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+ 
+These content items enable you retrieve, process and analyze email files, and manage phishing alerts . The out-of-the-box items are robust enough to get you started, but are easily customizable to fit your specific requirements.
+
+
+### Playbooks
+This content pack contains a playbook and two sub-playbooks.
+ 
+| Playbook | Description | Notes |
+|---------------- | ------------- | ------------- |
+| Phishing Alerts Investigation | This is the main playbook and the default playbook for the **Phishing Alerts** incident type. Use this playbook to investigate and remediate a potential phishing incident fetched from either your Email Security Gateway or through your SIEM. One of the playbook's main tasks is to retrieve the original email file from your Email Security Gateway or Email Service Provider. </br> The playbook's tasks include assessing the initial severity, processing results, and assessing the existence of similar phishing incidents in Cortex XSOAR. </br> No action is taken without an initial approval by the analyst using the playbook's inputs. | Under the playbook inputs, you can add the SOC email address to send the notifications via email. |
```
Comment (yaakovi) [inline content-repo/extra-docs/packs/Phishing-Alerts.md:24]:
```suggestion
| Phishing Alerts Investigation | This is the main playbook and the default playbook for the **Phishing Alerts** incident type. Use this playbook to investigate and remediate a potential phishing incident fetched from either your Email Security Gateway or through your SIEM. One of the playbook's main tasks is to retrieve the original email file from your Email Security Gateway or Email Service Provider. <br/> The playbook's tasks include assessing the initial severity, processing results, and assessing the existence of similar phishing incidents in Cortex XSOAR. <br/> No action is taken without an initial approval by the analyst using the playbook's inputs. | Under the playbook inputs, you can add the SOC email address to send the notifications via email. |
```

File: content-repo/extra-docs/packs/Phishing-Alerts.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+      * Email To
+      * Email CC
+      * Email Subject
+      * Email Headers
+      * Email Message ID (The original email message id and not the internal product id)
+      * Email Queue ID
+      * Email Source Domain (Optional)
+      * Occurred
+      * Severity
+      * UUID (For the product internal message/alert id) <br> For information about creating a mapper, see [Create a Mapper](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-admin/incidents/classification-and-mapping/create-a-mapper.html).
```
Comment (yaakovi) [inline content-repo/extra-docs/packs/Phishing-Alerts.md:94]:
```suggestion
      * UUID (For the product internal message/alert id) <br/> For information about creating a mapper, see [Create a Mapper](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-admin/incidents/classification-and-mapping/create-a-mapper.html).
```

File: content-repo/extra-docs/packs/Phishing-Alerts.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+* FireEye Email Security (EX)
+* Proofpoint TAP
+* Agari Phishing Defense
+* Mimecast
+
+SIEM products:
+* Splunk ES
+* QRadar
+ 
+ ### 2. Classification and Mapping
```
Comment (yaakovi) [inline content-repo/extra-docs/packs/Phishing-Alerts.md:69]:
```suggestion
#### 2. Classification and Mapping  
```

File: content-repo/extra-docs/packs/Phishing-Alerts.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+| SearchAndDeleteIntegration | Determines which product and playbook is used to search and delete the phishing email from users' inboxes.<br/> Set this to "O365" to use the O365 - Security And Compliance - Search And Delete playbook.<br/>Set this to "EWS" to use the Search And Delete Emails - EWS playbook. | EWS | Optional |
+| O365DeleteType | The method by which to delete emails using the O365 - Security And Compliance - Search And Delete playbook. Could be: "Soft" (recoverable), or "Hard" (unrecoverable\). Leave empty to decide manually for each email incident.<br/>This is only applicable if the SearchAndDeleteIntegration input is set to O365. | Soft | Optional |
+| O365DeleteTarget | The exchange location. Determines from where to search and delete emails searched using O365 playbooks. Use the values: </br> - "All" to search all mailboxes, </br> - "SingleMailbox" to search and delete the email only from the recipient's inbox, </br> - "Manual" to decide manually for every incident. </br> Note - searching all mailboxes may take a significant amount of time. This input is only applicable if the SearchAndDeleteIntegration input is set to O365. | SingleMailbox | Optional |
```
Comment (yaakovi) [inline content-repo/extra-docs/packs/Phishing-Alerts.md:129]:
```suggestion
| O365DeleteTarget | The exchange location. Determines from where to search and delete emails searched using O365 playbooks. Use the values: <br/> - "All" to search all mailboxes, <br/> - "SingleMailbox" to search and delete the email only from the recipient's inbox, <br/> - "Manual" to decide manually for every incident. <br/> Note - searching all mailboxes may take a significant amount of time. This input is only applicable if the SearchAndDeleteIntegration input is set to O365. | SingleMailbox | Optional |
```

**Comment by yaron-libman** [comment]
@altmannyarden Is there a reason this wasn't merged? Are we waiting on something?

**Comment by altmannyarden** [comment]
@yaron-libman We are waiting for to merge the content first :) so reason to have in our site if the content didn't merge yet.
https://github.com/demisto/content/pull/15777

**Comment by yaron-libman** [comment]
> @yaron-libman We are waiting to merge the content first :) no reason to have it our site if the content didn't merge yet. [demisto/content#15777](https://github.com/demisto/content/pull/15777)

That seems like a good reason.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**